2024.html

```diff
@@ -80,6 +80,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
             <li><a href="https://sarajoy.dev/">Sara Joy</a></li>
             <li><a href="https://emmajuettner.com/">Emma Juettner</a></li>
             <li><a href="https://eklausmeier.goip.de/blog">Elmar Klausmeier</a></li>
+            <li><a href="https://joelchrono.xyz/">joelchrono</a></li>            
```

Comment:
Please sort by alphabet (it seems before Sara)—thanks!

Reply:
should be done now